runtime/makelib/targets.mk.linux.inc.ftl

```diff
@@ -21,7 +21,16 @@
 -->
 
 <#if uma.spec.processor.arm>
-OBJCOPY := bcm2708hardfp-objcopy
+ARM_ARCH_FLAGS := -march=armv6 -marm -mfpu=vfp -mfloat-abi=hard
+<#--
```

Comment:
Can this comment be removed?

Reply:
Absolutely, it's an oversight. Thanks for pointing it out.